src/main/webapp/app/modeling-assessment-editor/modeling-assessment-editor.component.ts

```diff
@@ -163,6 +169,22 @@ export class ModelingAssessmentEditorComponent implements OnInit, OnDestroy {
         this.isLoading = false;
     }
 
+    private getComplaint(): void {
```

Comment:
This method should just receive the result id as a parameter.

Reply:
Done